src/main/java/io/stargate/sgv2/jsonapi/service/cqldriver/executor/NamespaceCache.java

```diff
@@ -113,4 +114,9 @@ private Uni<CollectionSettings> getVectorProperties(
               }
             });
   }
+
+  public void evictCollectionSettingCacheEntry(String collectionName) {
+    Log.error("evict the entry");
```

Comment:
Why this in log.eror, can you change it to info?

Reply:
... and in general, sounds more like `debug` or `trace` entry. And if used, should include some identifying info (evict which entry?).